### ISSUE TITLE
Project is empty after restarting Docker container for v1.0.0 image

### ISSUE THREAD

**Issue by scottpageindysoft** [node-red/node-red-docker#148]
### What are the steps to reproduce?
- Create a project
- Add nodes, create flows
- Add tabs, add nodes, create flows
- Use project history to push/pull for git
- Install a local package.json for development work on custom nodes
- Restart node-red Docker container

### What happens?
- I'm prompted to open a project
- I select the only listed project, to open
- My project is opened, and I don't see any of my flows, tabs, or dashboard.

### What do you expect to happen?
To see what I did, before I restarted the node-red Docker container

### Please tell us about your environment:
MacOS (job forced, not personal choice, can't change, hate it), Mojave 10.14.6
Docker Desktop:
- Engine: 19.03.2
- Compose: 1.24.1
- Kubernetes: 1.14.6
- Notary: 0.6.1
- Credential Helper: 0.6.3
- Machine: 0.16.2

- [ ] Node-RED version: 1.0.0
- [ ] Node.js version: 10.16.3
- [ ] npm version: 6.9.0
- [ ] Platform/OS: Docker - Alpine Linux - 3.9.4
- [ ] Browser: Chrome 77.0.3865.90 (Official Build) (64-bit)

**Comment by scottpageindysoft** [comment]
Appreciate the move to this project, I missed the node-red-docker one when I filed this.
I'll work on getting some more info.  I started a new container, new project, and have not since seen the issue.
It's quite possible that I goofed and contaminated the node-red root project, when I was using uibuilder and started a new vue project in /data/projects/<project_name>/uibuilder/<Vue_project>.

We can close this.  I'll open an issue if I see it again.
